src/main/scala/mesosphere/marathon/api/v2/AppsResource.scala

```diff
@@ -102,7 +103,7 @@ class AppsResource @Inject() (
       case _ =>
         Response
           .status(409)
-          .entity(Json.obj("message" -> s"An app with id [${app.id}] already exists.").toString)
+          .entity(Json.obj("message" -> s"An app with id [${app.id}] already exists.").toString())
```

Comment:
why `toString()`? Generally, I like the convention of reserving `()` to method invocations with side effects.

(But this is not important to me here, so if you disagree, leave it unchanged)